code/obj/item/clothing/chameleon_js.dm

```diff
@@ -1241,7 +1241,7 @@
 	var/scramble_prints = 0
 
 	insulated
-		desc = "These gloves are electrically insulated."
+		desc = "Tough synthleather work gloves styled in a high-visibility yellow color. They are electrically insulated, and provide full protection against most shocks."
```

Comment:
minor, but insuls would be synthrubber, not synthleather; for mat purposes and so people tracing down insuls aren't looking for leather fibres

Reply:
I'd thought about that, but wouldn't big rubber gloves be super awkward to work with? I'm not sure what best to put here

Reply:
big rubber gloves are super awkward; i always imagined electrician's gloves for the insuls, but i'm just thinking because electrical insulation is tied to the synthrubber property and it can cue in people when they might be trying to make their own gloves in the nanofab that they need synthrubber as a material in order to produce a pair